mjpc/agent.cc

```diff
@@ -73,6 +73,13 @@ void Agent::Initialize(const mjModel* model) {
   if (model_) mj_deleteModel(model_);
   model_ = mj_copyModel(nullptr, model);  // agent's copy of model
 
+  // check for limits on all actuators
+  for (int i = 0; i < model_->nu; i++) {
+    if (!model_->actuator_ctrllimited[i]) {
+      mju_error("Model is missing ctrl limits on at least one actuator.\n");
```

Comment:
Will it print a lot of the same errors, one for each actuator? 
Maybe we can also provide the index where it's missing. It will help for large models. 

@thowell

Reply:
Will add index printing for all missing actuators limits and then error.

Reply:
LGTM